index.html

```diff
@@ -6,28 +6,30 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
-    <title>Document</title>
+    <title>Portfolio Kazaneza</title>
 </head>
 <body>
 
-    
+    <header>
         <nav class="nav">
             <label class="logo">KAZANEZA GENTIL</label>
```

Comment:
As requested earlier please use a link tag `<a>` to enclose the logo like <label class="logo"><a href="home'>KAZANEZA GENTIL</a></label>, This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info